### PR TITLE
add cartridge

### PR DESCRIPTION
**Pull request by tarrencev**
adding cartridge to the mix :)

### CODE REVIEW

File: data/cartridge.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  "short_description": "Starknet Gaming Console",
+  "tags": ["Games"],
+  "contracts": [],
+  "goerliContracts": [],
+  "audits": [],
+  "dotw": false,
+  "links": {
+    "website": "https://cartridge.gg/",
+    "careers": "",
+    "twitter": "https://twitter.com/cartridge_gg",
```
Comment (ismaeldm95) [inline data/cartridge.json:13]:
```suggestion
    "twitter": "https://twitter.com/cartridge-gg",
```
Fixing github URL

File: data/cartridge.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    "telegram": "",
+    "discord": "https://discord.gg/cartridge",
+    "github": "https://github.com/cartridge_gg",
+    "youtube": "",
+    "medium": "",
+    "mirror": ""
+  },
+  "twitterName": "cartridge_gg",
+  "teamInfo": {
+    "founded": "2022-04-00T00:00:00.000Z",
```
Comment (ismaeldm95) [inline data/cartridge.json:23]:
```suggestion
    "founded": "2022-04-01T00:00:00.000Z",
```
Fixing founding date

File: data/cartridge.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    "anonymous": false
+  },
+  "tokens": [],
+  "media": {
+    "logoUrl": "https://static.cartridge.gg/logo.jpeg",
+    "bannerUrl": "https://static.cartridge.gg/banner.png",
+    "previewUrl": "https://static.cartridge.gg/preview.jpeg",
```
Comment (ismaeldm95) [inline data/cartridge.json:30]: In order for this to work, you'll need to add the domain [in next.config.js](https://github.com/argentlabs/dappland/blob/develop/next.config.js) like so:

```
  images: {
    domains: [
      "dv3jj1unlp2jl.cloudfront.net",
      "pbs.twimg.com",
      "github.com",
      "eykar.org",
      "drive.google.com",
      "mintsquare.sfo3.cdn.digitaloceanspaces.com",
      "www.frenslands.xyz",
      "raw.githubusercontent.com",
      "lh3.googleusercontent.com",
      "static.cartridge.gg" --> ADD THIS LINE
    ],
  }, 
```

**Review by ismaeldm95 — CHANGES_REQUESTED** [review]
Thanks for adding cartridge! happy to see it included 💪
I added a few comments to make it work

File: data/cartridge.json (hunk 1 above)
Comment (graemeblackwood) [reply data/cartridge.json:13]: @ismaeldm95 the correct Twitter handle is `cartridge_gg`

**Comment by tarrencev** [comment]
@ismaeldm95 thanks for your feedback! i've made the suggested changes 🚀

**Pull request merged**